package.json

```diff
@@ -56,5 +60,14 @@
                 "path": "./snippets/monkeyc.json"
             }
         ]
+    },
+    "scripts": {
+        "vscode:prepublish": "npm run compile",
+        "compile": "tsc",
+        "watch": "tsc -watch"
+    },
+    "devDependencies": {
```

Comment:
can you avoid importing .exe file? Can you use clang-format or clang-format-binaries packages instead?

https://github.com/angular/clang-format/blob/245ffa69d14f2be2b595853f33937db0bca4850c/index.js#L169-L186

https://github.com/lh2966/-sql/blob/c529be9f2427d7bd1b0fded23dca58019634f8cd/tools/gulp-tasks/format.js#L90

https://github.com/JJPro/atom-configs/blob/a68f311b98b266fb00f25f302173ca69e17356b1/packages/clang-format/lib/clang-format.js

https://github.com/marcosbaima/angular/blob/a4a423a083f42ee737aab0e4cc26d6d1177f822d/tools/gulp-tasks/format.js

Reply:
Made the necessary changes but I do not have Mac or Linux to test it.

Reply:
@Abasz Tested it on Mac. Works well. Thank you!